app/components/Events.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <StackLayout>
+        <StackLayout class="eventItem" v-for="e in events" >
+            <Label class="category" :text="e.category" width="100%"/>
+            <Label class="title" :text="e.title" width="100%"/>
+            <Label class="text description" :text="e.description" width="100%"/>
+            <Label class="spacer" />
+            <StackLayout  class="hbox" orientation="horizontal" >
+                <Label class="date textLeft" width="45%">D: {{ e.date }}</Label>
+                <Label class="location textRight" width="55%">L: {{ e.location}}</Label>
+            </StackLayout>
+            <Label class="text time" width="100%">Start: {{ moment( e.start_time ) }}</Label>
+            <StackLayout class="hbox text" orientation="horizontal">
+                <Label class="time textLeft" width="45%">End: {{ moment( e.end_time ) }}</Label>
+                <Label class="people textRight" width="55%">P: {{ e.people }}</Label>
+            </StackLayout>
+        </StackLayout>
+    </StackLayout>
+</template>
+
+<script>
+   import * as http from "http";
+   import moment from "moment";
+   export default {
+       data() {
+           return {
+               events: []
+           };
+       }, 
+       mounted() {
+           this.getEvents();
+           console.log( "Mounted: getEvents processed" );
+       },
+       methods: {
+           getEvents() {
+               http.getJSON( "https://pickup-app-backend.herokuapp.com/api/events" ).then( result => {
+                   this.events = result;
+                   console.log( "Results received" );
+                   } ), error => {
+                       console.log( "Could not get events" );
+               }
+           },
+           moment: function( time ) {
+               return moment(time).format( 'h:mm a' );
```

Comment:
we don't need spaces around the parenthesis

Reply:
Added space to moment(time) to match the rest.